actionlist/app/actions/action_items.js

```diff
@@ -0,0 +1,6 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+import { createAction } from 'redux-actions';
+import * as types from './types';
```

Comment:
Instead of importing the wildcard, you should destructure the types you'll need (since I'm presuming things like preference drawer action types will also go into `types.js` eventually).